back/app/models/custom_field.rb

```diff
@@ -85,7 +85,6 @@ class CustomField < ApplicationRecord
   before_validation :sanitize_description_multiloc
   after_create(if: :domicile?) { Area.recreate_custom_field_options }
 
-  scope :with_resource_type, ->(resource_type) { where(resource_type: resource_type) }
   scope :registration, -> { where(resource_type: 'User') }
```

Comment:
Would this scope be better named `user`? More obvious then that it applies to the `User` resource type

Reply:
I tend to have a preference for "registration fields" over "user fields" because I feel it's more clear what it's about and less abstract. I did a quick search on Slack ("registration fields"), and it seems that other colleagues also use this term to talk about fields for zipcode, gender etc.

We also attempted to clarify terminology in [this document](https://www.notion.so/citizenlab/Custom-field-specifications-a1342b71b1f64a8eaefd796bbc8144f1). I tried to adopt the terminology that Koen DH was working out (I think he pushed for the term "registration fields" back then).

Actually, I also have my doubts if we should set `'User'` as resource type. It feels a bit messy to use the mechanisms for polymorphism for this (providing the type, but not the ID). Usually we represent the global context as "nil" for both type and ID.